.github/workflows/trivy_periodic_image_scan.yml

```diff
@@ -13,20 +13,21 @@ on:
     - cron: "0 0 * * *"
 
 jobs:
-  trivy-matrix:
-    name: periodic scan
+  lower-case:
```

Comment:
it's not apparent to me why this is even needed.  could you please add some comments describing the purpose for this?